apps/src/logToCloud.js

```diff
@@ -33,6 +39,11 @@ module.exports = {
       return;
     }
 
+    if (Math.random() > sampleRate) {
```

Comment:
If sampleRate is 0.5, then we would be logging on all numbers from 0.00 to 0.50, including one extra float (the 0.50 float) since the range of Math.random() is [0.0, 1.0), I believe. Feel free to ignore, this doesn't matter at all.